db/migrate/20201112214758_create_contact_attempts.rb

```diff
@@ -0,0 +1,43 @@
+class CreateContactAttempts < ActiveRecord::Migration[6.0]
+  def up
+    create_table :contact_attempts do |t|
+      t.references :patient, index: true
+      t.references :user, index: true
+
+      t.boolean :successful, index: true
+      t.string :note
+
+      t.timestamps
+    end
+
+    # mysql does not support full joins
+    execute <<-SQL.squish
+      INSERT INTO contact_attempts (patient_id, user_id, successful, note, created_at, updated_at)
+      SELECT histories.patient_id as patient_id,
+            users.id as user_id,
+            histories.comment NOT LIKE '%unsuccessful%' as successful,
```

Comment:
Similar note to before these changes: it's possible someone used the word "unsuccessful" in their successful contact attempt in the additional note they leave. Maybe make this even more precise: "unsuccessful contact attempt".

Reply:
yeah i thought about that too and I think it makes sense to make this more precise. I just wanted to also check to see if this comment has ever been something other than "unsuccessful contact attempt"

Reply:
fixed here: https://github.com/SaraAlert/SaraAlert/pull/517/commits/d4255790fcde36b945ebe22f3837f3eb65b886e4